nevergrad/functions/control/core.py

```diff
@@ -152,3 +159,12 @@ class Humanoid(BaseFunction):
07, -0.5945929428933572, 0.3587232211841977, -0.4104791162839761, 1.5986149190475882, 5.204750377221477, -0.26852433811945536, 0.19813165370374033, -0.3062371808686244, 0.004238222556072027, -2.2209658318223635, 5.742622248085341, 0.6722949974624708, -0.500485113914267, -0.3062371808686244, 0.004238222556072027, -2.2209658318223635, 5.742622248085341, 0.6722949974624708, -0.500485113914267, 0.26012708391359357, 0.08467556295331613, -0.21391009904140584, 5.3210057409790155, 0.03506536347635228, -0.001275684097233967, 0.042059501206005254, -0.23778440296675274, -0.1226403571516149, 5.348137526673431, -0.043272898300492156, 0.06022962996263737, 0.2937088892684222, -0.1977541092634194, -0.28966619577855046, 5.298852322727817, 0.23447012164868147, -0.025647992690023803, 0.3599616142316438, 0.2572452733614258, -0.10288112745228332, 5.19810241527611, 0.20728458029872535, -0.10683192755398696, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.29556968580778875, -8.260898269152916, 5.587670071623525, 8.924550388476465, 7.307887810310615, -5.797082988253691, 21.76326786049523, 0.5281607939082064, 3.0078721315484658, -8.25644817359464, 12.92236085784975, -3.07346302346624, -0.15972201114587897, 3.8488401400073924, 2.9013015503137547, -3.0030744277640964, -4.125822702844248, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, -0.0005411157531993445, 0.00018425900752185883, -7.056479603492841e-05, 0.0004850329710863729, 0.0013067914565368577, -0.0003632249271021157, -1.499782615828459e-05, 1.5396507342996525e-05, -8.412078417064285e-07, 0.00010005310427990925, 0.000202125250115722, -0.00023887949816504833, 2.358707586858282, -2.038571404847552, -0.4050387837681951, 4.231544859434408, -2.40103925025192, 33.864309172326074, -0.5579333098944332, 0.34076320959977724, -0.08857964550304018, -0.9922056623081122, -1.3998733333878197, 1.0435800890962725, -2.0834375475277884, 1.2667537566732037, -0.30076659164163855, -2.936598660670857, -4.701568329244183, 0.5482891196808485, -10.245820936980875, 7.420004642248826, 0.4255468058894504, 17.167740585723312, 16.175268727949796, 181.48592716060185, 0.11325750561423298, -0.06582083133392769, 0.0526578564471078, 0.2371962961790206, 0.3042510288306412, -0.12644331987411908, 2.523881380476937, -1.547423424434062, 0.33845447817037794, 3.7044012690867003, 5.77270922236946, -1.4549487373900374, 8.414220368253863, -4.740154811733693, 0.37906093062842394, -6.603153276131259, -14.672811393696326, 152.19174733969905, -9.948554129284157e-07, -3.1729733223247383e-07, 1.6278277777934387e-07, -3.6080215121179876e-05, -4.327504944517283e-05, 0.00011092777930590747, 0.0005553154635918675, -0.00021156350367889417, 7.217738550388757e-05, -0.0007368680078812343, -0.001578912969766702, 0.0007730670670347712, -7.128808629768361e-07, 4.5728191857655706e-07, 2.2865347644914397e-07, 9.111110122544389e-06, 1.160719855911783e-05, 9.381773874831127e-06, 5.569740868840155e-06, -1.4891833711621733e-06, 4.342784980667928e-06, 6.777033412499121e-05, 8.971821928675218e-05, 3.889718928290386e-05]
```

Comment:
copying something like this is never good for maintenance, and actually completely useless since there are multiple ways to use the original one

Reply:
Created an inherited class.

Reply:
Please do not create another class, otherwise making combinations (with noise for instance) or propagating to other cases (not just Humanoids) will become a complete pain. If it's a parameter of the base class, then it becomes available for **all** Mujoco cases, which is much cleaner.